ui/apps/platform/cypress/integration/networkGraph/connections.test.js

```diff
@@ -19,7 +19,7 @@ describe('Network Graph connections filter', () => {
     const activeSubstring = 'active connection';
     const allowedSubstring = 'allowed connection';
 
-    it('active appears in namespace edge tooltip', () => {
+    it.skip('active appears in namespace edge tooltip', () => {
```

Comment:
You could add `.skip` to the whole describe block.

Reply:
🤦